addons/payment_stripe/models/payment_acquirer.py

```diff
@@ -13,6 +13,8 @@ class PaymentAcquirerStripe(models.Model):
 
     stripe_secret_key = fields.Char(required_if_provider='stripe')
     stripe_publishable_key = fields.Char(required_if_provider='stripe')
+    image_url = fields.Char("Image URL", required_if_provider='stripe',
```

Comment:
Ratio is important and there is no need to check anything in Odoo as is part of the Stripe profile setup anyway. 

This module is about integrating with an external system. As this external system has a mechanism to deal with the image and the URL is actually listed under `data-image`  [here](https://stripe.com/docs/checkout) and we explain it in the help tip (see below on line 17) I think is very easy for the user to cut/paste the URL.

I see two other alternatives:
- find a way to change aspect ratio of company logo 
- store one more binary field in this module and store the **square** company logo.